src/robusta/integrations/git/git_repo.py

```diff
@@ -26,6 +27,20 @@ class GitRepoManager:
     manager_lock = threading.Lock()
     repo_map = defaultdict(None)
 
+    @staticmethod
+    def setup_host_keys(custom_host_keys: List[str]):
+        if not os.path.exists(SSH_ROOT_DIR):
+            os.mkdir(SSH_ROOT_DIR)
+        with open(f"{SSH_ROOT_DIR}/known_hosts", "w") as f:
```

Comment:
Reviewers note: we are recreating the `known_hosts` file here from scratch. This is ran at the start of the runner and on each subsequent config change that might affect the list of known hosts. Is this correct? I think so, as there are no other places in the codebase where `known_hosts` are modified and as far as I understand, with a new deployment it is not even created in the first place.